modules/tables/src/lib/table/table-types.d.ts

```diff
@@ -1,4 +1,4 @@
-export type Field = Date | Float32Array | String;
+export type Field = any;
```

Comment:
This was causing a TS error where the table Field was inconsistent with the Arrow field.